testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/ExpressionSupportSmokeTestCase.java

```diff
@@ -273,6 +273,11 @@ private void organizeAttributes(PathAddress address, ModelNode description, Mode
                 }
                 boolean conflict = false;
                 for (String baseAttr : base) {
+                    if (!resource.hasDefined(baseAttr)) {
+                        conflict = true;
+                        break;
+                    }
```

Comment:
This is just a workaround, but it's ok for now. https://issues.jboss.org/browse/WFLY-8085 is for a (hopefully) better fix.